src/__tests__/definitions/invalid-payload-template.asl.json

```diff
@@ -0,0 +1,16 @@
+{
+  "Comment": "This SFN Should Fail Validation but does not",
```

Comment:
Maybe explain why it is failing

Reply:
was copy/pasted from the bug report. I updated